bundler.d/ovirt.rb

```diff
@@ -1,3 +1,4 @@
 group :ovirt do
-  gem 'fog-ovirt', '~> 1.2.0'
+  # gem 'fog-ovirt', '~> 1.2.0'
+  gem 'fog-ovirt', :path => '../fog-ovirt'
```

Comment:
probably not what you planned to commit?

Reply:
@tbrisker sorry! fixed it :)